docs/sphinx/examples/python/tutorials/vqe.ipynb

```diff
@@ -42,28 +42,25 @@
     "import cudaq\n",
     "\n",
     "geometry = [('H', (0., 0., 0.)), ('H', (0., 0., .7474))]\n",
-    "\n",
     "molecule, data = cudaq.chemistry.create_molecular_hamiltonian(\n",
     "    geometry, 'sto-3g', 1, 0)\n",
     "\n",
     "electron_count = data.n_electrons\n",
     "qubit_count = 2 * data.n_orbitals\n",
     "\n",
-    "kernel, angles = cudaq.make_kernel(list)\n",
-    "qubits = kernel.qalloc(qubit_count)\n",
-    "\n",
-    "# Prepare the Hartree Fock State.\n",
-    "kernel.x(qubits[0])\n",
-    "kernel.x(qubits[1])\n",
+    "@cudaq.kernel(jit=True)\n",
+    "def kernel(qubit_count: int, angles:list[float]):\n",
+    "    qubits = cudaq.qvector(qubit_count)\n",
     "\n",
-    "# Adds parameterized gates based on the UCCSD ansatz.\n",
-    "cudaq.kernels.uccsd(kernel, qubits, angles, electron_count, qubit_count)\n",
+    "    # Prepare the Hartree Fock State.\n",
+    "    x(qubits[0])\n",
+    "    x(qubits[1])\n",
     "\n",
-    "parameter_count = cudaq.kernels.uccsd_num_parameters(electron_count,\n",
-    "                                                     qubit_count)\n",
+    "    # Adds parameterized gates based on the UCCSD ansatz.\n",
+    "    cudaq.uccsd(qubits, angles, electron_count, qubit_count)\n",
```

Comment:
This needs fixing; the path seems to have changed compared to the original one. I think it may be nice to migrate all uccsd functions under cudaq.chemistry, but 1) we need to properly deprecate to original paths and give a comprehensive message for what to change and 2) uccsd and uccsd_num_parameters should be in the same location.